zerver/lib/actions.py

```diff
@@ -4394,20 +4394,29 @@ def get_email_gateway_message_string_from_address(address: str) -> Optional[str]
 
     return msg_string
 
-def decode_email_address(email: str) -> Optional[Tuple[str, str]]:
-    # Perform the reverse of encode_email_address. Returns a tuple of (streamname, email_token)
+def decode_email_address(email: str) -> Optional[Tuple[str, str, bool]]:
+    # Perform the reverse of encode_email_address. Returns a tuple of
+    # (streamname, email_token, show_sender)
     msg_string = get_email_gateway_message_string_from_address(email)
+    show_sender = False
 
     if msg_string is None:
         return None
     elif '.' in msg_string:
         # Workaround for Google Groups and other programs that don't accept emails
```

Comment:
unrelated to this PR, but given that we fully control the email, why don't we make the email always use a `-` or `.`?